README.md

```diff
@@ -88,11 +101,11 @@ loop for (3)
 }
 ```
 
-More information regarding the syntax can be found on the [Night website](https://night-website.dynamicsquid.repl.co/html/reference.html).
+More information regarding the syntax can be found on the [Night Website](https://night-website.dynamicsquid.repl.co/html/reference.html).
 
 ### Timeline
 
-A timeline of Night, both past and future versions. Each new release, the code is scrapped and started anew (cause I have no idea what I'm doing).
+A timeline of Night, both past and future versions. Each new release, the code is scrapped and started afresh (cause I have no idea what I'm doing).
```

Comment:
"afresh" sounds weird, like Night is something you buy at the grocery store lol

Reply:
![image](https://user-images.githubusercontent.com/33234914/97471896-1fae0900-1941-11eb-9e72-2bf2e6a37d26.png)

Reply:
so you win